src/ui/overlay.js

```diff
@@ -48,5 +45,16 @@ const refreshChaosRecipe = async () => {
 };
 
 ipcRenderer.on('forceChaosRecipeRefresh', () => refreshChaosRecipe());
-setInterval(() => refreshChaosRecipe(), POLLING_DELAY);
+setInterval(() => refreshChaosRecipe(), settings.get('overlay.refreshTime') * 1000 || 60000);
```

Comment:
I would keep a `DEFAULT_REFRESH_TIME` constant, so we can do 
`(settings.get('overlay.refreshTime') || DEFAULT_REFRESH_TIME) * 1000`

Reply:
Nice idea,  it's more user friendly like this :)
I don't know why I can't commit on this change, so if u can do it that's nice :)